src/app/devices/deviceContent/components/deviceContentNav.tsx

```diff
@@ -46,41 +45,34 @@ export default class DeviceContentNavComponent extends React.Component<DeviceCon
         const url = this.props.match.url;
 
         const navItems = isEdgeDevice ? NAV_LINK_ITEMS : NAV_LINK_ITEMS_NONEDGE;
-        const nonPnpNavLinks = navItems.map((nav: string) => ({
+        let navLinks = navItems.map((nav: string) => ({
             key: nav,
             name: context.t((ResourceKeys.deviceContent.navBar as any)[nav]), // tslint:disable-line:no-any
             url: `#${url}/${nav}/?${ROUTE_PARAMS.DEVICE_ID}=${encodeURIComponent(deviceId)}`
         }));
 
         const groups: INavLinkGroup[] = [];
-        groups.push({
-            links: nonPnpNavLinks,
-            name: context.t(ResourceKeys.deviceContent.navBar.nonpnp)
-        });
 
         if (!isEdgeDevice && isPnPDevice) {
             const pnpNavLinks = [{
                 key: NAV_LINK_ITEM_PNP,
                 name: context.t((ResourceKeys.deviceContent.navBar as any)[NAV_LINK_ITEM_PNP]), // tslint:disable-line:no-any
                 url: `#${url}/${NAV_LINK_ITEM_PNP}/?${ROUTE_PARAMS.DEVICE_ID}=${encodeURIComponent(deviceId)}`
             }];
-            groups.push({
-                links: pnpNavLinks,
-                name: context.t(ResourceKeys.deviceContent.navBar.pnp),
-            });
+
+            navLinks = [...navLinks, ...pnpNavLinks];
```

Comment:
since pnpNavLinks is a single item, line 48 can be `const navLinks  =...` and this line can be `navLinks.push(pnpNavLinks);`